bigbluebutton-html5/imports/ui/components/recording/container.jsx

```diff
@@ -8,17 +8,21 @@ import RecordingComponent from './component';
 
 const RecordingContainer = props => <RecordingComponent {...props} />;
 
-export default withModalMounter(withTracker(({ mountModal }) => ({
-  closeModal() {
-    mountModal(null);
-  },
+export default withModalMounter(withTracker(({ mountModal }) => {
+  const meeting = Meetings.findOne({ meetingId: Auth.meetingID });
```

Comment:
I don't think you need the whole meeting object.

`const { recording, time } = Meetings.findOne({ meetingId: Auth.meetingID }).recordProp;`